hollow/src/main/java/com/netflix/hollow/api/producer/HollowProducer.java

```diff
@@ -678,7 +678,7 @@ private void applyDelta(Blob blob, HollowReadStateEngine stateEngine) throws IOE
     private void validate(HollowProducer.ReadState readState) {
     	com.netflix.hollow.api.producer.HollowProducerListener.ProducerStatus.Builder psb = listeners.fireValidationStart(readState);
     	List<Throwable> exceptions = new ArrayList<>();
-    	AllValidationStatusBuilder valStatus = AllValidationStatus.builder();
+    	OverAllValidationStatusBuilder valStatus = OverAllValidationStatus.builder();
```

Comment:
Small nitpick, I think overall is one word, so `OverallValidation...` instead?

Reply:
Sounds good. Thank you.